public/modules/custom/grants_handler/src/ApplicationHandler.php

```diff
@@ -1721,7 +1721,7 @@ public static function getCompanyApplications(
       $searchParams = [
         'service' => 'AvustushakemusIntegraatio',
         'business_id' => $selectedCompany['identifier'],
-        'lookfor' => $lookForAppEnv,
+        'lookfor' => $lookForAppEnv . ',applicant_type:' . $selectedRoleData['type'],
```

Comment:
Mä lisäsin ton roolin tohon hakuun ku yritin debuggaa miks hakijan hakemussivu on tyhjänä. Ikävä kyl en nyt pysty taas keskittymään tähän niin kauaa että löytäsin syyn. Enkä ees tiedä johtuuko mistään mitä täs pullaris tapahtuu mut tässä mä sen näin enkä hannaa mergee ennen ku palikat toimii.

Mut siis tää lomake toimii kuten odotettua.

Reply:
@jiisuominen En kyllä usko että tää tähän liittyis, mutta voin yrittää kurkata :D

Reply:
Mä mergeen ja tehään tosta uus tiketti.